### PR TITLE
Practice react

### PR DESCRIPTION
**Pull request by Erikaguilera123**
## Sumary

The first training, the scenary og Sing up

### CODE REVIEW

File: cypress/const/helpers.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
 export function HAVETEXT (elemnt, text) {
-cy.get(`#${elemnt}`).type(text).should('have.value', text);
+cy.get(elemnt).type(text).should('have.value', text);
+}
```
Comment (betoobernal) [inline cypress/const/helpers.js]: indentation.

File: cypress/integration/registroAngular.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    cy.get(IDs.btnSingUp).click();
+
+    cy.get(`.${IDs.errorRegister}`).contains(VAR.emailInvalid);
+    cy.get(`.${IDs.errorRegister}`).contains(VAR.passInvalid);
+    cy.get(`.${IDs.errorRegister}`).contains(VAR.userInvalid);
+
+  });
+
+  it('Sign up', () => { 
+
```
Comment (betoobernal) [inline cypress/integration/registroAngular.spec.js]: remove space

File: cypress/integration/registroAngular.spec.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  it('Sign up', () => { 
+    
+   HAVETEXT(IDs.name,VAR.name);
+   HAVETEXT(IDs.email,VAR.email);
+   HAVETEXT(IDs.password,VAR.pass);
+
+   cy.get(IDs.btnSingUp).click();
+
+   cy.get(IDs.nameProfile).should('be.visible');
+
```
Comment (betoobernal) [inline cypress/integration/registroAngular.spec.js]: remove space

File: cypress/integration/registroAngular.spec.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+context('Register', () => {
+  beforeEach(() => {
+    cy.visit(URL.index);
+    cy.get(IDs.idSingUp).click();
+  });
+
+  it('Sign up validate null', () => { 
+    cy.get(IDs.btnSingUp).should('not.have.class', 'disabled');
+
```
Comment (betoobernal) [inline cypress/integration/registroAngular.spec.js]: remove space

File: cypress/page-objects/registroAngular.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+
+import { VAR } from '../const/constants';
+
+export const IDs = {
+  idSingIn: 'a[href*="#login"]',
+  idSingUp:'a[href*="/register"]',
+  name:'input[formcontrolname*="username"',
+  email:'input[formcontrolname*="email"',
+  password: 'input[formcontrolname*="password"',
```
Comment (betoobernal) [inline cypress/page-objects/registroAngular.js]:
```suggestion
  name: 'input[formcontrolname*="username"]',
  email: 'input[formcontrolname*="email"]',
  password: 'input[formcontrolname*="password"]',
```

add spaces between `: '` and add `]` to selector

File: cypress/const/constants.js [hunk 6]
```diff
@@ -1,8 +1,15 @@
...
+
 export const VAR = {
-  email: 'correo@email.co',
-  textSmall: 'Small',
-  emailError: 'Invalid email address.'
+  email: `correo${Math.floor(Math.random() * 100) + 1}@email.co`,
+  name: `name${Math.floor(Math.random() * 100) + 1}`,
+  textSmall: 'Small10',
+  pass: 'a1b2c3d4e5',
+  nulo:' ',
```
Comment (betoobernal) [inline cypress/const/constants.js:8]: English*

File: cypress/const/constants.js [hunk 7]
```diff
@@ -1,8 +1,15 @@
+import{ RANDOM } from '../const/helpers';
+
 export const VAR = {
-  email: 'correo@email.co',
-  textSmall: 'Small',
-  emailError: 'Invalid email address.'
+  email: `correo${Math.floor(Math.random() * 100) + 1}@email.co`,
+  name: `name${Math.floor(Math.random() * 100) + 1}`,
+  textSmall: 'Small10',
+  pass: 'a1b2c3d4e5',
```
Comment (betoobernal) [inline cypress/const/constants.js]: password*

File: cypress/integration/registroAngular.spec.js (hunk 2 above)
Comment (Erikaguilera123) [reply cypress/integration/registroAngular.spec.js]: ok

File: cypress/integration/registroAngular.spec.js (hunk 3 above)
Comment (Erikaguilera123) [reply cypress/integration/registroAngular.spec.js]: ok

File: cypress/integration/registroAngular.spec.js (hunk 4 above)
Comment (Erikaguilera123) [reply cypress/integration/registroAngular.spec.js]: ok

File: cypress/page-objects/registroAngular.js (hunk 5 above)
Comment (Erikaguilera123) [reply cypress/page-objects/registroAngular.js]: ok

File: cypress/const/constants.js (hunk 7 above)
Comment (Erikaguilera123) [reply cypress/const/constants.js]: ok

File: cypress/const/helpers.js (hunk 1 above)
Comment (Erikaguilera123) [reply cypress/const/helpers.js]: ok tab

File: cypress/integration/registroAngular.spec.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+    cy.get(`.${IDs.errorRegister}`).contains(VAR.userInvalid);
+  });
+
+  it('Sign up', () => { 
+   HAVETEXT(IDs.name,VAR.name);
+   HAVETEXT(IDs.email,VAR.email);
+   HAVETEXT(IDs.password,VAR.password);
+
+   cy.get(IDs.btnSingUp).click();
+   cy.get(IDs.nameProfile).should('be.visible');
```
Comment (eledwinn) [inline cypress/integration/registroAngular.spec.js]: Fix indentation, It should have four spaces

File: cypress/integration/registroAngular.spec.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import { VAR, URL } from '../const/constants';
```
Comment (eledwinn) [inline cypress/integration/registroAngular.spec.js]: Use english in file names like `registroAngular.spec.js`.

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: cypress/const/helpers.js [hunk 10]
```diff
@@ -1,3 +1,7 @@
 export function HAVETEXT (elemnt, text) {
-cy.get(`#${elemnt}`).type(text).should('have.value', text);
+  cy.get(elemnt).type(text).should('have.value', text);
+}
+
+export function RANDOM (){
+  Math.floor(Math.random() * 100) + 1;
 }
```
Comment (betoobernal) [inline cypress/const/helpers.js]: use a more descriptive name. `RANDOM_NUMBER` for example.

**Review by eledwinn — APPROVED** [review]

**Comment by Erikaguilera123** [comment]
this is my master branch of Angular project

**Pull request closed without merging**